pkg/endpoints/endpoints.go

```diff
@@ -142,3 +143,18 @@ func FetchRCEndpoint(serviceEndpoint []ServiceEndpoint) string {
 	}
 	return ""
 }
+
+// CostructRegionFromZone Calculate region based on location/zone.
+func CostructRegionFromZone(zone string) string {
```

Comment:
is this same logic been used in the powervs sdk?

Reply:
Yes.